tests/expectations/metrics/test_metric_providers.py

```diff
@@ -29,17 +34,26 @@
 from great_expectations.expectations.metrics.table_metric_provider import (
     TableMetricProvider,
 )
-from great_expectations.expectations.registry import (
-    _registered_metrics,
-)
+from great_expectations.expectations import registry
 
 
-def test__base_metric_provider__registration():
-    """This tests whether the MetricProvider class registers the correct metrics."""
+_REGISTERED_METRICS_ORIGINAL: dict = copy.deepcopy(registry._registered_metrics)
```

Comment:
You shouldn't need this global.
You can do this in the setup section of a fixture.

Reply:
@Kilo59 Please take a look at the latest -- thank you.